scraper/code_gov/models.py

```diff
@@ -520,12 +524,8 @@ def from_doecode(klass, record):
         status = record.get("ever_announced")
         if status is None:
             raise ValueError('DOE CODE: Unable to determine "ever_announced" value!')
-        elif status:
-            status = "Production"
-        else:
-            status = "Development"
 
-        project["status"] = status
+        project["status"] = "Production" if status else "Development"
```

Comment:
Is this one of the pylint recommendations?

I actually was discussing this difference in style earlier this week with a colleague.

Reply:
So `pylint`'s flag and remediation recommendation is: `Unnecessary "elif" after "raise", remove the leading "el" from "elif" (no-else-raise)`. The change you marked was just my personal style coming out and I'm happy to explicitly follow the `pylint` remediation if you prefer. I just tend to use the `x if condition else y` flow for simple assignments as I find it cleaner.